runtime/src/iree/hal/utils/libmpi_test.cc

```diff
@@ -11,7 +11,9 @@
 #include "iree/testing/gtest.h"
 #include "iree/testing/status_matchers.h"
 
-namespace {
+namespace iree {
```

Comment:
tests are defined in anonymous namespaces to avoid conflicts - everything in the file should be static to it and putting it in a namespace implies that it's exported.